src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/Footer.tsx

```diff
@@ -27,7 +28,11 @@ export const Footer: React.FC = () => {
     <>
       <Box px={[2, 4, 12]} mb={[4, 6, 6]}>
         <Text variant={["lg-display", "xl", "xxl"]} pr={0} textAlign="center">
-          Meet your new advisor. It’s Artsy.
+          Meet your new advisor.{" "}
+          <Media at="xs">
```

Comment:
**suggestion:** I would suggest to avoid using Media here and instead use display none for bigger devices